### DISCUSSION TITLE
plz give me the deets.

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#1067] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
im a dum lil kid who knows nothingg about compuuters. plz help with cryptoSMITE!!! idk wut anything means (i checked the website AND writeup)

**Comment by Blobby-Boi** [comment]
read

**Reply by Averagedude69420** [reply, marked as answer]
go here-https://dl.osu.bio/ it has easy to follow instructions with prebuilt shims

**Comment by ghost** [comment]
thanks!
